samples/msgext-northwind-inventory-csharp/lab/Exercise 01 - Set up.md

```diff
@@ -0,0 +1,95 @@
+
+Open your browser and browse to Microsoft 365 ([https://www.office.com/](https://www.office.com/)). Log in using the Microsoft 365 account you will be using throughout the lab. You may be asked to set up multi-factor authentication.
+
+Using the "waffle" menu in the upper left corner of the page 1️⃣ , navigate to the OneDrive application within Microsoft 365 2️⃣ .
+
+![Navigating to the OneDrive application in Microsoft 365](./images/01-02-CopySampleFiles-01.png)
+
+Within OneDrive, navigate to "My Files" 1️⃣ . If there's a documents folder, click into that as well. If not, you can work directly within the "My Files" location.
+
+![Navigating to your documents in OneDrive](./images/01-02-CopySampleFiles-02.png)
+
+Now click "Add new" 1️⃣ and "Folder" 2️⃣ to create a new folder.
+
+![Adding a new folder in OneDrive](./images/01-02-CopySampleFiles-03.png)
+
+Name the folder "Northwind contracts" and click "Create".
+
+![Naming the new folder "Northwind contracts"](./images/01-02-CopySampleFiles-03b.png)
+
+Now, from within this new folder, click "Add new" 1️⃣  again but this time click "Files upload" 2️⃣ .
+
+![Adding new files to the new folder](./images/01-02-CopySampleFiles-04.png)
+
+Now browse to the **sampleDocs** folder within your working folder. Highlight all the files 1️⃣ and click "OK" 2️⃣  to upload them all.
+
+![Uploading the sample files from this repo into the folder](./images/01-02-CopySampleFiles-05.png)
+
+By doing this step early, there's a good chance that the Microsoft 365 search engine will have discovered them by the time you're ready for them.
+
+## Step 4 - Install and set up Teams Toolkit for Visual Studio Code
+
+Install Teams Toolkit for Visual Studio [Teams Toolkit extension](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+
+![Installing Teams Toolkit in Visual Studio](./images/01-04-visual-studio-install.png)
+
+In the debug dropdown menu of Visual Studio, select Dev Tunnels > Create A Tunnel (set authentication type to Public) or select an existing public dev tunnel.
+
+![Create dev tunnel](./images/01-04-create-devtunnel-01.png)
+
+![Create dev tunnel](./images/01-04-create-devtunnel-02.png)
+
+In the debug dropdown menu of Visual Studio, select default startup project > **Microsoft Teams (browser)**
+
+![select debug profile](./images/01-04-debug-dropdown.png)
+
+In Visual Studio, right-click your **TeamsApp** project and **Select Teams Toolkit > Prepare Teams App Dependencies**
+
+![Prepare Teams App Dependencies](./images/01-04-prepare-dependencies-01.png)
+
+Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps.
```

Comment:
This process also prompts the developer to create a resource group where Azure resources are provisioned to, we should include this in the steps.

Reply:
Fixed